frontend/index.html

```diff
@@ -43,9 +43,12 @@ <h1 id="headline">Pencil<span style="color: brown">2</span>Pixel</h1>
             <button id="dropbtn">Select Style</button>
             <div id="dropdown-content">
                 <a id="nostyle" onclick="changeStyle(0)">No Style</a>
-                <a id="style1" onclick="changeStyle(1)">Style 1</a>
-                <a id="style2" onclick="changeStyle(2)">Style 2</a>
-                <a id="style3" onclick="changeStyle(3)">Style 3</a>
+                <a id="style1" onclick="changeStyle(1)">Photography</a>
```

Comment:
You could change the id name, so it's clear which style is passed in the switch case in script.js

Reply:
Makes sense, I changed the id names to their corresponding style